types/underscore/index.d.ts

```diff
@@ -733,30 +733,33 @@ declare module _ {
         ): TypeOfList<V>[];
 
         /**
-        * Computes the union of the passed-in arrays: the list of unique items, in order, that are
-        * present in one or more of the arrays.
-        * @param arrays Array of arrays to compute the union of.
-        * @return The union of elements within `arrays`.
-        **/
-        union<T>(...arrays: _.List<T>[]): T[];
+         * Computes the union of the passed-in `lists`: the list of unique
+         * items, in order, that are present in one or more of the lists.
+         * @param lists The lists to compute the union of.
+         * @returns The union of elements within `lists`.
+         **/
+        union<T>(...lists: List<T>[]): T[];
 
         /**
-        * Computes the list of values that are the intersection of all the arrays. Each value in the result
-        * is present in each of the arrays.
-        * @param arrays Array of arrays to compute the intersection of.
-        * @return The intersection of elements within `arrays`.
-        **/
-        intersection<T>(...arrays: _.List<T>[]): T[];
+         * Computes the list of values that are the intersection of the
+         * passed-in `lists`. Each value in the result is present in each of
+         * the lists.
+         * @param lists The lists to compute the intersection of.
+         * @returns The intersection of elements within the `lists`.
+         **/
+        intersection<T>(...lists: List<T>[]): T[];
 
         /**
-        * Similar to without, but returns the values from array that are not present in the other arrays.
-        * @param array Keeps values that are within `others`.
-        * @param others The values to keep within `array`.
-        * @return Copy of `array` with only `others` values.
-        **/
+         * Similar to without, but returns the values from `list` that are not
+         * present in `others`.
+         * @param list The starting list.
+         * @param others The values to exclude from `list`.
```

Comment:
```suggestion
         * @param others Zero or more lists containing the values to exclude from `list`.
```

Reply:
I think I'll shorten this slightly to just "The lists of values" instead of specifying zero or more, mostly because I'm not a huge fan of highlighting zero as an option. Let me know if that works for you.

Reply:
Sure, no problem.